src/App.jsx

```diff
@@ -41,14 +46,20 @@ const App = () => {
   }, [restore]);
 
   const [selectedUser, setSelectedUser] = useState('');
-  /** @type {useState<userListObject[]>} */
-  const [userList, setUserList] = useState([]);
+  /** @type {userListObject[]} */
+  const initialUserList = [];
+  const [userList, setUserList] = useState(initialUserList);
   const [loadingUsers, setLoadingUsers] = useState(false);
   const [loadingActive, setLoadingActive] = useState(false);
 
   const selectedUserObject = useMemo(() => ({ selectedUser, setSelectedUser }), [selectedUser]);
   const userListObject = useMemo(() => ({ userList, setUserList }), [userList]);
 
+  /** @type {inboxListObject[]} */
+  const initialInboxList = [];
```

Comment:
I don't think this variable `initialInboxList` is being used anywhere else (maybe I missed it though.)

If it isn't, I don't think we need it. Just make line 51: `const [userList, setUserList] = useState([]);`

Reply:
It's there for typing purposes since we're not using TypeScript.

With how it's initialized here, we'll get:
<img width="938" alt="Screen Shot 2023-05-19 at 14 37 59" src="https://github.com/codeforpdx/PASS/assets/14917816/069d064c-e9f5-4a55-ad29-aa6b18fb8822">

Without it, we'll get:
<img width="832" alt="Screen Shot 2023-05-19 at 14 36 35" src="https://github.com/codeforpdx/PASS/assets/14917816/7656298c-fca3-4f37-9490-8ac799a5d543">
With never[] as the type.